spec/web_helpers.rb

```diff
@@ -0,0 +1,28 @@
+module Helpers
+
+  def sign_up(email, password)
+    visit('/')
+    click_link('Sign up')
+    fill_in(:user_email, with: email)
+    fill_in(:user_password, with: password)
+    fill_in(:user_password_confirmation, with: password)
+    click_button('Sign up')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.